src/pages/AssetsItem/components/About/components/Links/Links.js

```diff
@@ -0,0 +1,81 @@
+// @flow strict
+
+import React from 'react'
+import { withI18n } from '@lingui/react'
+import { type I18n as I18nType } from '@lingui/core'
+
+import {
+  JIcon,
+  JLink,
+} from 'components/base'
+
+import styles from './links.m.scss'
+
+type OwnProps = {|
+  +assetPage: ?DigitalAssetPage,
+|}
+
+type Props = {|
+  ...OwnProps,
+  +i18n: I18nType,
+|}
+
+function getURLLabel(
+  i18n: I18nType,
+  type: string,
+): string {
+  switch (type) {
+    case 'site':
+      return i18n._(
+        'AssetsItem.About.Links.label.site',
+        null,
+        { defaults: 'Token Website' },
+      )
+
+    default:
+      return i18n._(
+        'AssetsItem.About.Links.label.info',
+        { type },
+        { defaults: '{type} Info' },
+      )
+  }
+}
+
+function Links({
+  i18n,
+  assetPage,
+}: Props) {
+  if (!assetPage) {
+    return null
+  }
+
+  return (
+    <ul className={styles.core}>
+      {assetPage.urls.map((item: DigitalAssetPageUrl) => (
+        <li
+          key={item.url}
+          className={styles.link}
+        >
+          <JLink
+            href={item.url}
+            className={styles.anchor}
+            target='_blank'
```

Comment:
`JLink` sets these attributes to every external link by default.

https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/components/base/JLink/JLink.js#L65-L74

I think, we should actually separate JLink into multiple components. Something like:
- RouterLink – for internal links and accept routeName (and proxy other Route props), not path
- ExternalLink – for external links with href and `_blank noopener noreferrer` by default
- TrustedLink – for external links we trust (like privacy policy, or maybe Jwallet landing)

The latter may be not needed, actually.

And then we share styles between them.

I think, this will remove a lot of confusion.

Reply:
Not in this PR, of course.